client/components/SingleUser.js

```diff
@@ -100,9 +55,10 @@ class SingleUser extends Component {
               <img
               src="https://i0.wp.com/www.thisblogrules.com/wp-content/uploads/2010/02/batman-for-facebook.jpg?resize=250%2C280" alt="" />
             </CardMedia>
-          </div>
-          <div className="userInfo">
-            <CardTitle title={"Name: "}  subtitle="Card subtitle" />
+
+            <CardTitle
+            title={ `Name: ${userInfo.displayName}` }
```

Comment:
So for React JSX we don't put a space after the curly brace.
https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/jsx-curly-spacing.md
But for objects we normally put a space after the curly brace.
https://eslint.org/docs/rules/object-curly-spacing